src/components/book-card/book-card.js

```diff
@@ -5,11 +5,12 @@ import BookProgress from '../book-progress/book-progress';
 
 export default function BookCard(props) {
   const {
-    percent, chapter, genre, title, author,
+    id, percent, chapter, genre, title, author,
```

Comment:
Please ensure that your `list of books` displays books from `Redux store` (by using `useSelector` function) according to the project requirement.

![image](https://user-images.githubusercontent.com/10332499/160613819-f7982a1b-7d82-4d10-af4c-aa26a5d56834.png)

Please check [HERE for more details](https://react-redux.js.org/api/hooks).

Reply:
The `List of books` is being generated using the `Redux store`

Please see the following [books.js](https://github.com/Sboursen/BookstoreCMS/blob/using-redux-in-react-components/src/routes/books/books.js) page.